sdks/python/container/py310/base_image_requirements.txt

```diff
@@ -154,4 +155,5 @@ urllib3==1.26.13
 websocket-client==1.4.2
 Werkzeug==2.2.2
 wrapt==1.14.1
+xgboost==1.7.1
```

Comment:
I would like to push back on adding these dependencies to container because they would increase the size of the container image since these are not dependencies of Apache Beam